packages/cra-template/package.json

```diff
@@ -27,7 +27,7 @@
     "template.json"
   ],
   "devDependencies": {
-    "@coveo/headless": "latest",
+    "@coveo/headless": "alpha",
```

Comment:
This will not affect the generated project.
You need to update `packages/cra-template/template.json`
[The template.json file](https://create-react-app.dev/docs/custom-templates/#the-templatejson-file).

Unless, you find a way to read from the package.json of the template...